content/hardware/05.pro-solutions/solutions-and-kits/opta/tutorials/getting-started-connectivity/content.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Using BLE, WiFi and Ethernet on the Arduino Opta'
+description: "Learn how to make use of the Arduino Opta's connectivity features"
+difficulty: beginner
+tags:
+  - Wi-Fi®
+  - Ethernet
+  - Bluetooth®
+  - Connectivity
+author: 'Benjamin Dannegård'
+libraries:
+  - name: Arduino BLE
+    url: https://www.arduino.cc/reference/en/libraries/arduinoble/
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+hardware:
+  - hardware/05.pro-solutions/solutions-and-kits/opta
+---
+
+## Overview
+
+The Arduino® Opta is a powerful PLC device that has many features, allowing you to customize its use for your solution. Among these features are the standard connectivity features: Wi-Fi®, Ethernet and Bluetooth®. In this tutorial we will go through how to use these features with the Arduino IDE and the Arduino Opta.
+
+## Goals
+
+- Learn how to use Bluetooth® Low Energy on the Opta
+- Learn how to use Wi-Fi® on the Opta
+- Learn how to use Ethernet on the Opta
+
+### Required Hardware and Software
+
+- Arduino Opta
+- Ethernet cable
+- [Arduino IDE](https://www.arduino.cc/en/software)
+- USB-C® cable (either USB-C to USB-A or USB-C to USB-C)
+- Wi-Fi® Access Point with Internet Access
+
+## Instructions
+
+Using the Arduino IDE we can easily work with these peripherals. Some features have their own library that we can make use of. First we need to install the appropriate core for the Arduino Opta, go into the **Board manager** and search for **Opta Mbed core**. When the latest version of the core is installed, we can move on to try out the connectivity options.
+
+### Wi-Fi®
+
+The files necessary for using the Wi-Fi® on the Opta are included in the core. To try it out, we can go to **File > Examples**, and under **Examples for Opta** we can find the **WiFi** section. Inside there is an example sketch called **WiFiWebClient**. Open this example sketch and fill out the WiFi details in the **arduino_secrets.h** tab. The sketch will make the Opta connect to desired website defined within `char server[] = "example.com";` variable. If the connection is successful, it will then print the websites' HTML content to the serial monitor. If the sketch ran successfully, the output should look like the image below.
```

Comment:
```suggestion
The files necessary for using the Wi-Fi® on the Arduino Opta® are included in the core. To try it out, we can go to **File > Examples**, and under **Examples for Opta** we can find the **WiFi** section. Inside there is an example sketch called **WiFiWebClient**. Open this example sketch and fill out the WiFi details in the **arduino_secrets.h** tab. The sketch will make the  Arduino Opta® connect to desired website defined within `char server[] = "example.com";` variable. If the connection is successful, it will then print the websites' HTML content to the serial monitor. If the sketch ran successfully, the output should look like the image below.
```
Trademark addition for Arduino Opta® (Recent addition) and given new rule, we need to use full name instead of only Opta.